server/api/v1/emojis/[id]/index.get.ts

```diff
@@ -0,0 +1,38 @@
+import path from "path";
+import module from "module";
+import fs from "fs/promises";
+import sanitize from "sanitize-filename";
+
+// // Get current file path (for ESM)
+// const __filename = fileURLToPath(import.meta.url);
+// const __dirname = path.dirname(__filename);
+
+// const require = createRequire(import.meta.url);
+// const nodeModulesPath = path.dirname(require.resolve("lodash/package.json"));
+
+const twemojiJson = module.findPackageJSON(
+  "@discordapp/twemoji",
+  import.meta.url,
+);
+
+export default defineEventHandler(async (h3) => {
+  if (!twemojiJson)
+    throw createError({
+      statusCode: 500,
+      statusMessage: "Failed to resolve emoji package",
+    });
+
```

Comment:
Maybe add some sort of authentication for this route? I don't think we need a separate ACL for it, so if you just pass no ACLs to the aclManager, it'll allow sessions and tokens.